apps/mobile/src/components/DrawerNavigation/Sidebar.tsx

```diff
@@ -0,0 +1,53 @@
+import type { FC } from 'react';
+import { View } from 'react-native';
+import type { DrawerContentComponentProps } from '@react-navigation/drawer';
+import DashboardNavigator from '@walless/app/features/DashboardLayout/Navigator';
+import {
+	useSafeAreaInsets,
+	useSnapshot,
+	useWidgets,
+} from '@walless/app/utils/hooks';
```

Comment:
don't use this kind of path, import from `@walless/app` instead - if anything not available yet at package level, go export it from within the package.